docs/user_guide/concepts/main_concepts/data_management.rst

```diff
@@ -174,3 +174,75 @@ For example, it is theoretically possible to use S3 ``s3://`` for metadata and G
+
+Example
+~~~~~~~
+
+Let's consider a simple example where we have some tasks that needs to operate huge dataframes.
+
+The first task reads a file from the object store, shuffles the data, save to local disk and passes the path to the next task.
+
+.. code-block:: python
+    @task()
+    def task_remove_column(input_file: FlyteFile, column_name: str) -> FlyteFile:
+        """
+        Reads the input file as a DataFrame, removes a specified column, and outputs it as a new file.
+        """
+        input_file.download()
+        df = pd.read_csv(input_file.path)
+
+        # remove column
+        if column_name in df.columns:
+            df = df.drop(columns=[column_name])
+
+        output_file_path = "data_finished.csv"
+        df.to_csv(output_file_path, index=False)
+
+        return FlyteFile(output_file_path)
+       ...
+
+The second task reads the file from the previous task, removes a column, save to local disk and return the path.
+
+.. code-block:: python
+    @task()
+    def task_remove_column(input_file: FlyteFile, column_name: str) -> FlyteFile:
+        """
+        Reads the input file as a DataFrame, removes a specified column, and outputs it as a new file.
+        """
+        input_file.download()
+        df = pd.read_csv(input_file.path)
+
+        # remove column
+        if column_name in df.columns:
+            df = df.drop(columns=[column_name])
+
+        output_file_path = "data_finished.csv"
+        df.to_csv(output_file_path, index=False)
+
+        return FlyteFile(output_file_path)
+       ...
+
+And this is how the workflow looks like:
+
+.. code-block:: python
+    @workflow
+    def wf() -> FlyteFile:
+        # In Flyte sandbox, existing files must be accessed within the workflow, not directly in a task.
+        existed_file = FlyteFile("s3://custom-bucket/data.csv")
+        shuffled_file = task_read_and_shuffle_file(input_file=existed_file)
+        result_file = task_remove_column(input_file=shuffled_file, column_name="County")
+        return result_file
+       ...
+
+This example shows how to access an existing file in a MinIO bucket from the Flyte Sandbox and pass it between tasks with ``FlyteFile``.
+
+* Files in MinIO must be accessed at the workflow level in Flyte Sandbox, then passed to tasks to avoid permission errors.
+* When a workflow outputs a local file as a ``FlyteFile``, Flyte automatically uploads it to MinIO and provides an S3 URL for downstream tasks, no manual uploads needed. Take a look at the following:
+
+First task output metadata:
+
+.. image:: https://raw.githubusercontent.com/flyteorg/static-resources/main/flyte/concepts/data_movement/flyte_data_movement_example_output.png
+
+Second task input metadata:
+
+.. image:: https://raw.githubusercontent.com/flyteorg/static-resources/main/flyte/concepts/data_movement/flyte_data_movement_example_input.png
```

Comment:
use gitsha first, then change it back.
gitsha: 
https://github.com/flyteorg/static-resources/blob/ddec386ab74897e3c09d89fdaf305e77a2ecc52c/flyte/concepts/data_movement/flyte_data_movement_example_input.png

Reply:
719db18af583f727d5b3c099e01d031a22a3216b fixed